### ISSUE TITLE
No OperatorHub Items Found

### ISSUE THREAD

**Issue by devopskarthikeyan** [code-ready/crc#1243]
### General information

  * OS: Windows 10
  * Hypervisor:Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)?No

## CRC version
```
PS C:\Users\Tagitdemo2> oc version
Client Version: 4.3.19-202005020952-4fb2d4d
Server Version: 4.4.3
Kubernetes Version: v1.17.1
```
```
PS C:\Users\Tagitdemo2> crc status
CRC VM:          Running
OpenShift:       Running (v4.4.3)
Disk Usage:      12.2GB of 32.72GB (Inside the CRC VM)
Cache Usage:     14.27GB
Cache Directory: C:\Users\Tagitdemo2\.crc\cache
```
```
PS C:\Users\Tagitdemo2> oc get co
NAME                                       VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
authentication                             4.4.3     True        False         False      16d
cloud-credential                           4.4.3     True        False         False      16d
cluster-autoscaler                         4.4.3     True        False         False      16d
console                                    4.4.3     True        False         False      16d
csi-snapshot-controller                    4.4.3     True        False         False      16d
dns                                        4.4.3     True        False         False      6m40s
etcd                                       4.4.3     True        False         False      16d
image-registry                             4.4.3     True        False         False      16d
ingress                                    4.4.3     True        False         False      16d
insights                                   4.4.3     True        False         False      16d
kube-apiserver                             4.4.3     True        False         False      16d
kube-controller-manager                    4.4.3     True        False         False      16d
kube-scheduler                             4.4.3     True        False         False      16d
kube-storage-version-migrator              4.4.3     True        False         False      16d
machine-api                                4.4.3     True        False         False      16d
machine-config                             4.4.3     True        False         False      16d
marketplace                                4.4.3     True        False         False      6m35s
monitoring                                 4.4.3     True        False         False      16d
network                                    4.4.3     True        False         False      16d
node-tuning                                4.4.3     True        False         False      16d
openshift-apiserver                        4.4.3     True        False         False      5m13s
openshift-controller-manager               4.4.3     True        False         False      15d
openshift-samples                          4.4.3     True        False         False      3s
operator-lifecycle-manager                 4.4.3     True        False         False      16d
operator-lifecycle-manager-catalog         4.4.3     True        False         False      16d
operator-lifecycle-manager-packageserver   4.4.3     True        False         False      6m21s
service-ca                                 4.4.3     True        False         False      16d
service-catalog-apiserver                  4.4.3     True        False         False      16d
service-catalog-controller-manager         4.4.3     True        False         False      16d
storage                                    4.4.3     True        False         False      16d


PS C:\Users\Tagitdemo2> oc get opsrc -n openshift-marketplace
NAME                  TYPE          ENDPOINT              REGISTRY              DISPLAYNAME           PUBLISHER   STATUS        MESSAGE                                                                                            AGE
certified-operators   appregistry   https://quay.io/cnr   certified-operators   Certified Operators   Red Hat     Configuring   Get https://quay.io/cnr/api/v1/packages?namespace=certified-operators: context deadline exceeded   16d
community-operators   appregistry   https://quay.io/cnr   community-operators   Community Operators   Red Hat     Configuring   Get https://quay.io/cnr/api/v1/packages?namespace=community-operators: context deadline exceeded   16d
redhat-marketplace    appregistry   https://quay.io/cnr   redhat-marketplace    Red Hat Marketplace   Red Hat     Configuring   Get https://quay.io/cnr/api/v1/packages?namespace=redhat-marketplace: context deadline exceeded    16d
redhat-operators      appregistry   https://quay.io/cnr   redhat-operators      Red Hat Operators     Red Hat     Configuring   Get https://quay.io/cnr/api/v1/packages?namespace=redhat-operators: context deadline exceeded      16d
```
tried to crc delete and set up new CRC often times but not able to resolve the issues. Please check and help on this.

Thanks,
P.Karthikeyan

**Comment by gbraad** [comment]
Remove "Windows 10" from the title as this is unrelated to the OS; this is based on the cluster image

**Comment by gbraad** [comment]
Closing as duplicate of #1162

**Comment by gbraad** [comment]
As per https://github.com/code-ready/crc/issues/1162#issuecomment-615900632 his might be related to a failing start. Please try again after deleting the cluster
